index.js

```diff
@@ -0,0 +1,105 @@
+import {
+  displayBooks,
+  displayForm,
+  contact,
+  displayContact,
+  showBooksOnly,
+  showOnlyContact,
+  showOnlyForm,
+} from "./modules/view.js";
+
+import { DateTime } from "./modules/luxon.js";
+
+const displayDate = document.querySelector(".date");
+
+displayDate.innerHTML = `${DateTime.local().toLocaleString(
+  DateTime.DATETIME_MED_WITH_SECONDS
+)}`;
```

Comment:
- [ ] The date is not rendering on the UI. You can try something like += and add the date. I am just suggesting though but In know once you dig further, you can fit it.
### What I have here
![image](https://user-images.githubusercontent.com/86200356/205587559-f8f2466b-a951-4c99-9d16-77659cebadca.png)